install_scripts/fits.sh

```diff
@@ -7,13 +7,16 @@ if [ -f "$SHARED_DIR/install_scripts/config" ]; then
   . $SHARED_DIR/install_scripts/config
 fi
 
-DOWNLOAD_URL="http://projects.iq.harvard.edu/files/fits/files/fits-${FITS_VERSION}.zip"
-cd $DOWNLOAD_DIR
-if [ ! -f "fits.zip" ]; then
-  sudo curl $DOWNLOAD_URL > fits.zip
+if [ ! -d fits-$FITS_VERSION ]; then
+  DOWNLOAD_URL="http://projects.iq.harvard.edu/files/fits/files/fits-${FITS_VERSION}.zip"
+  cd $DOWNLOAD_DIR
+  if [ ! -f "fits.zip" ]; then
+    curl $DOWNLOAD_URL -o fits.zip
+  fi
+  unzip fits.zip
+  chmod a+x fits-$FITS_VERSION/*.sh
+  FITS_PATH="${DOWNLOAD_DIR}/fits-${FITS_VERSION}"
+  cd
+  echo "PATH=\${PATH}:$FITS_PATH" >> .bashrc
```

Comment:
When I rebuild from scratch, fits.sh isn't in the path of the `ubuntu` user, and the line isn't at the end of their .bashrc.

The Vagrantfile only has `privileged: false` for the `demo-app.sh` script (https://github.com/projecthydra-labs/hydra-vagrant/blob/d7d79c82ed0d484bba0a54325b81993d108cddb0/Vagrantfile#L27), so I think the others are all being run as root.